packages/master-detail-layout/src/vaadin-master-detail-layout.js

```diff
@@ -389,14 +394,14 @@ class MasterDetailLayout extends SlotStylesMixin(ResizeMixin(ElementMixin(Themab
   render() {
     return html`
       <div part="backdrop" @click="${this.__onBackdropClick}"></div>
-      <div id="master" part="master" ?inert="${this._hasDetail && this._overlay && this.containment === 'layout'}">
+      <div id="master" part="master" ?inert="${this._hasDetail && this._drawer && this.containment === 'layout'}">
```

Comment:
Unrelated, but I wonder why we don't always make the master area inert in overlay mode. Regardless whether you use drawer, stack, viewport or layout containment, the master side should not be interactable or focusable?

Reply:
IIRC the idea was that `aria-modal` should indicate that other parts should not be interactable but that can be only used with the "viewport" containment. At least that's how I understood the Rolf's idea.

We can probably change to always set `inert` for consistency as it might be more reliable.